### ISSUE TITLE
Add Cockpit ZFS Manager

### ISSUE THREAD

**Issue by exterrestris** [exterrestris/home-network#17]
Cockpit does not currently have built-in support for ZFS - https://github.com/optimans/cockpit-zfs-manager adds this

**Comment by exterrestris** [comment]
Must be manually installed by cloning repo. Possibly depends on #15?

**Comment by exterrestris** [comment]
Module asks for configuration on first run - custom role should write the config file

**Comment by exterrestris** [comment]
Merged into main in 69a8f26ea1e969a3adbffb8fa98fdbb20b9012d2

**Comment by exterrestris** [comment]
Config will be handled in #33
